sdk/storage/Azure.Storage.Common/src/Azure.Storage.Common.csproj

```diff
@@ -17,16 +17,15 @@
       Microsoft Azure Storage REST API Reference - https://docs.microsoft.com/en-us/rest/api/storageservices/
     </Description>
     <RootNamespace>Azure.Storage</RootNamespace>
+    <IncludeOperationsSharedSource>true</IncludeOperationsSharedSource>
```

Comment:
Why do we need this?

Reply:
to pull in the common files

Reply:
there were some new chain dependencies that came about as a result of the request context / classification changes anne made